pkg/api/testapi/testapi.go

```diff
@@ -200,3 +200,7 @@ func GetCodecForObject(obj runtime.Object) (runtime.Codec, error) {
 	}
 	return nil, fmt.Errorf("unexpected kind: %v", kind)
 }
+
+func NewTestGroup(external, internal unversioned.GroupVersion) TestGroup {
+	return TestGroup{external, internal}
+}
```

Comment:
The two fields of TestGroup are private. When I construct the TestGroup using struct literals in my test the compiler complains.